CalculatorLibrary/Calculator.cs

```diff
@@ -0,0 +1,81 @@
+﻿using CalculatorLibrary.Models;
+
+namespace CalculatorLibrary;
+
+public class Calculator
+{
+    private List<Equation> equations;
+    private string filePath;
+    public int counter = 0;    // Number of times the Calculator has been used
+
+    public Calculator(List<Equation> equations, string filePath)
```

Comment:
If you change the arguments names you can get rid of the ugly "this" 😁
![image](https://user-images.githubusercontent.com/108711476/214431701-d0fa58bc-4f70-4311-9303-17e49cab7541.png)

Reply:
I saw 2 ways of doing it. One was with underscore and the other with "this". I preferred the latter and it didn't bothered me at all. I honestly consider things like "_equations" even uglier :)

Reply:
@kraven88 yeah at the end of the day it's your personal preference. In enterprise you'll follow whatever code standard you company has 🤓😁